org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties

```diff
@@ -39,6 +39,8 @@ AdvancedNewProjectPage_projEclipseSDKFeature=Create Feature
 AdvancedNewProjectPage_projEclipseP2=Create Update Site
 AdvancedNewProjectPage_projIde=Generic IDE Support
 AdvancedNewProjectPage_projIde_description=This option is mandatory for any IDE integration or for use as a language server.\nAn additional project with suffix .ide will be created.
+AdvancedNewProjectPage_languageServer=Build Language Server Jar
```

Comment:
Can we stick to `JAR` instead of `Jar`? Just to be consistent with the naming used in Eclipse.
(At least JAR is being used in both export and new wizards in general;)